reservations/migrations/0003_from_data_to_main_fields.py

```diff
@@ -0,0 +1,127 @@
+# Generated by Django 2.1.2 on 2019-01-17 07:13
+from decimal import Decimal, InvalidOperation
+from functools import reduce
+
+from django.db import IntegrityError, migrations, transaction
+
+CHAR_FIELDS_TO_DATA_KEY_MAP = {
+    'phone_number': "applicant.contact.mobile_phone",
+    'address': "applicant.postal.street_address",
+    'zip_code': "applicant.postal.postal_code",
+    'municipality': "applicant.postal.munacipality",
```

Comment:
There seems to be a typo in "applicant.postal.mun**A**cipality".

Reply:
yeah, frontend had a typo there, so all the reservations now have this typo too 🙂